internal/server/v1/firehose/actions.go

```diff
@@ -27,7 +26,7 @@ const (
 )
 
 func (api *firehoseAPI) handleReset(w http.ResponseWriter, r *http.Request) {
-	var reqBody entropyKafka.ResetParams
+	var reqBody entropy.ResetParams
```

Comment:
@ishanarya0 there were dependency issues when importing entropy to dex.

It would be nice if we can use the model directly from entropy, but it seems like it brings more issues because as we are importing entropy to dex, it pulls all the dependencies that entropy uses which most of them are not required by dex.

So for now, dex will define it's own entropy models.